source/content/guides/drupal-9-commerce.md

```diff
@@ -42,10 +42,10 @@ This guide covers installing [Drupal Commerce](https://drupalcommerce.org/), an
 
 ## Create a New Drupal 9 Site
 
-1. Use the Terminus Build Tools plugin to create a new Drupal 9 site from the Pantheon [Drupal Recommended](https://github.com/pantheon-systems/drupal-recommended) repository on GitHub:
+1. Use the Terminus Build Tools plugin to create a new Drupal 9 site from the Pantheon [Drupal Recommended](https://github.com/pantheon-systems/drupal-composer-managed) repository on GitHub:
 
  ```bash{promptUser: user}
- terminus build:project:create pantheon-systems/example-drupal-recommended $SITENAME
+ terminus build:project:create pantheon-systems/d9 $SITENAME
```

Comment:
Neither pantheon-systems/example-drupal-recommended nor pantheon-systems/d9 exist. We could just use the `d9` alias (without `pantheon-systems/`) if we made the corresponding change in build tools.

Reply:
This is what [Kevin had said](https://github.com/pantheon-systems/documentation/pull/7292#discussion_r856370306) as well, though I didn't realize `pantheon-systems/` needed to be removed. Fixed in [ee5baa4](https://github.com/pantheon-systems/documentation/pull/7292/commits/ee5baa4dd3fdff3db779761a2ce75c4d2b7e2bca).